### ISSUE TITLE
aws-c2a fails to install at 0.7.1

### ISSUE THREAD

**Issue by bilalq** [cdklabs/awscdk-change-analyzer#57]
I was really excited when I found https://constructs.dev/packages/@aws-c2a/cdk-pipelines-step/v/0.7.1?lang=typescript.

However, when I tried to use it in a pipeline, the `Check` step fails at the build phase of the codebuild run:

```
[Container] 2021/12/03 15:33:35 Entering phase BUILD
--
22 | [Container] 2021/12/03 15:33:35 Running command set -e
23 |  
24 | [Container] 2021/12/03 15:33:35 Running command [ -z "${RULE_SET}" ] \|\| aws s3 cp s3://$BUCKET/$RULE_SET $RULE_SET
25 |  
26 | [Container] 2021/12/03 15:33:35 Running command npm install -g aws-c2a
27 | npm ERR! code ETARGET
28 | npm ERR! notarget No matching version found for @aws-c2a/visualizer@^0.7.1.
29 | npm ERR! notarget In most cases you or one of your dependencies are requesting
30 | npm ERR! notarget a package version that doesn't exist.
31 | npm ERR! notarget
32 | npm ERR! notarget It was specified as a dependency of 'aws-c2a'
33 | npm ERR! notarget
34 |  
35 | npm ERR! A complete log of this run can be found in:
36 | npm ERR!     /root/.npm/_logs/2021-12-03T15_33_40_673Z-debug.log
37 |  
38 | [Container] 2021/12/03 15:33:40 Command did not exit successfully npm install -g aws-c2a exit status 1
39 | [Container] 2021/12/03 15:33:40 Phase complete: BUILD State: FAILED
40 | [Container] 2021/12/03 15:33:40 Phase context status code: COMMAND_EXECUTION_ERROR Message: Error while executing command: npm install -g aws-c2a. Reason: exit status 1
```

I was able to reproduce this locally on my machine as well:

```
[~/.../experiments] ➔ npm i -g aws-c2a
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @aws-c2a/visualizer@^0.7.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bilalq/.npm/_logs/2021-12-03T15_38_29_234Z-debug.log
```

Seems like some of the sibling packages didn't get published? Installing 0.7.0 directly works.

**Comment by bilalq** [comment]
Since the codebuild step just runs `npm install -g aws-c2a` without trying to match the version of `@aws-c2a/cdk-pipelines-step`, pinning just that dep in one's own package.json doesn't work either.

**Comment by rix0rrr** [comment]
Thanks! Fixed in `0.7.3`.
